src/pages/patientView/clinicalInformation/lib/convertSamplesData.spec.js

```diff
@@ -4,7 +4,7 @@ import {assert} from "chai";
 import {size} from "lodash";
 
 describe('', () => {
-    it('api data is properly transformed into table data', () => {
+    it.skip('api data is properly transformed into table data', () => {
```

Comment:
probably i shouldn't have rebased until we were ready to merge PR so it would be clear i was pushing to PR